tarocfg/server.go

```diff
@@ -208,9 +208,6 @@ func CreateServerFromConfig(cfg *Config, cfgLogger btclog.Logger,
 			TaroAddrBook: tarodbAddrBook,
 		},
 	})
-	if err != nil {
-		return nil, fmt.Errorf("unable to start server: %v", err)
-	}
 
 	return server, nil
```

Comment:
we could return directly where we define server, but given that it is a long function call (config is defined in-place) I left it here.